GiniVision/Classes/GiniVisionDocument.swift

```diff
@@ -12,27 +12,31 @@ import Foundation
  * Document processed by the _GiniVision_ library.
  */
 
-@objc public protocol GiniVisionDocument:class {
-    var type:GiniVisionDocumentType { get }
-    var data:Data { get }
-    var previewImage:UIImage? { get }
-    var isReviewable:Bool { get }
-    var isImported:Bool { get }
+@objc public protocol GiniVisionDocument: class {
+    var type: GiniVisionDocumentType { get }
+    var data: Data { get }
+    var previewImage: UIImage? { get }
+    var isReviewable: Bool { get }
+    var isImported: Bool { get }
     
     func checkType() throws
 }
 
 // MARK: GiniVisionDocumentType
 
-@objc public enum GiniVisionDocumentType:Int {
+@objc public enum GiniVisionDocumentType: Int {
     case pdf = 0
     case image = 1
+    case qrcode = 2
 }
 
 // MARK: GiniVisionDocumentBuilder
 
 /**
- * The `GiniVisionDocumentBuilder` provides a way to build a `GiniVisionDocument` from a `Data` object. The `DocumentSource` must be provided in the initialization, being optional but highly recommended setting the `DocumentImportMethod` afterwards. This could be an example of how a `GiniVisionDocument` should be built when it has been imported with the _Open with_ feature.
+ The `GiniVisionDocumentBuilder` provides a way to build a `GiniVisionDocument` from a `Data` object.
+ The `DocumentSource` must be provided in the initialization, being optional but highly recommended
+ setting the `DocumentImportMethod` afterwards. This could be an example of how a `GiniVisionDocument`
+ should be built when it has been imported with the _Open with_ feature.
```

Comment:
I really don't understand what you mean with these several sentences. Can you paraphrase?